stubs/fpdf2/@tests/stubtest_allowlist.txt

```diff
@@ -10,3 +10,6 @@ fpdf.fonts.HarfBuzzFont
 
 # Stubtest wants us to use Literals, but that is unreasonable.
 fpdf.unicode_script.UNICODE_RANGE_TO_SCRIPT
+
+# Ignore stubtest weirdness "fpdf.fonts.Glyph._DT is not present at runtime"
+fpdf.fonts.Glyph._DT
```

Comment:
looks like a mypy issue relating to https://github.com/python/mypy/blob/5c87e972d3c2c3c1f6229e1a655c4b903207dd32/mypy/plugins/dataclasses.py#L86 (at a guess, it's probably synthesizing a TypeVar and attaching it to the class body, then not ignoring the synthesized symbol when running stubtest?)

Reply:
we should probably report it upstream to mypy as a stubtest bug

Reply:
python/mypy#18811